### ISSUE TITLE
Block re-adding a tag for 1-hour after removal

### ISSUE THREAD

**Issue by qdacsvx** [iftechfoundation/ifdb-suggestion-tracker#113]
This is to prevent edit wars.

Although 1-hour isn't long, it has to be considered that multiple people (i.e. innocent people, not the perpetrator[s] who added the tag in the first place) may want to add the tag innocently who would be inconvenienced by a long guard period.

**Comment by dfabulich** [comment]
I don't think IFDB has any edit-war prevention functionality.

**Comment by qdacsvx** [comment]
No, hence this suggestion! Time-based blocks for general edits? Troublesome if you are working with another editor.

Perhaps there should be a slider for the admin to change the period of the protection 1-12 hours.

If users could remove tags #6, I would recommend a longer period for protection from deletion (by someone of the same privilege class) after addition: 12-24 hours. This is so one person can defend tags on their game pages against a group of attackers.
